backend/.gitignore

```diff
@@ -1,2 +1,3 @@
 tmp/
-.env
\ No newline at end of file
+.env
+backend
```

Comment:
Just double checking, is backend being added into the .gitignore?

Reply:
that's the backend binary file when you run go build it's not any go file no worries :)